packages/design-picker/src/components/premium-badge/index.tsx

```diff
@@ -45,20 +47,41 @@ const PremiumBadge: FunctionComponent< Props > = ( {
 	const divRef = useRef( null );
 	const [ isPopoverVisible, setIsPopoverVisible ] = useState( false );
 	const [ isHovered, setIsHovered ] = useState( false );
+	const [ isFocused, setIsFocused ] = useState( false );
+
 	return (
 		<div
 			className={ classNames( 'premium-badge', className, {
 				'premium-badge__compact-animation': shouldCompactWithAnimation,
 				'premium-badge--compact': shouldCompactWithAnimation && ! isHovered,
+				'premium-badge--is-focusable': shouldShowTooltipOnFocus,
 			} ) }
 			ref={ divRef }
+			role={ shouldShowTooltipOnFocus ? 'button' : undefined }
```

Comment:
Is there any better role for the badge? The button is a bit weird as it's not clickable 😓

Reply:
Hmmm... how about [note](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/note_role)? Or we could remove the role and add an eslint ignore rule. WDYT?